pkg/execute/executor.go

```diff
@@ -132,11 +132,13 @@ func runNotifierCommand(args []string) string {
 		}
 		if args[1] == "start" {
 			config.Notify = true
-			return "Notifier started!"
+			log.Logger.Info("Notifier enabled")
+			return "Brace yourselves, notifications are coming."
```

Comment:
Define constants for these strings at package level. We can easily change text later if needed.
Same for all text message being sent to slackc